analysis/metric_space.lean

```diff
@@ -386,5 +386,30 @@ have tendsto (λp:α×α, dist p.1 p.2) (nhds (a, b)) (nhds (dist a b)),
   from continuous_iff_tendsto.mp continuous_dist' (a, b),
 (hf.prod_mk hg).comp (by rw [nhds_prod_eq] at this; exact this)
 
+lemma tendsto_iff_dist_tendsto_zero { X Y : Type*} [metric_space X] [metric_space Y]
```

Comment:
Please reuse `α` and `β` from the head of the file.
Indent the second line and the content between `begin` and `end`.